llvm/test/CodeGen/X86/fp-intrinsics.ll

```diff
@@ -1,4 +1,4 @@
-; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --function fatan2 --version 2
```

Comment:
Did we use the right arguments for this test check generation? Don’t think this line should be here.